config/clusters/jupyter-meets-the-earth/common.values.yaml

```diff
@@ -321,9 +321,9 @@ dask-gateway:
                     # A default image is suggested via DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE env variable
                     "image": options.image,
                     "scheduler_extra_pod_labels": extra_labels,
-                    "scheduler_extra_pod_annotations": extra_annotations,
+                    "scheduler_extra_pod_annotations": scheduler_extra_pod_annotations,
                     "worker_extra_pod_labels": extra_labels,
-                    "worker_extra_pod_annotations": extra_annotations,
+                    "worker_extra_pod_annotations": scheduler_extra_pod_annotations,
```

Comment:
Out of curiosity, any reason to still keep the worker pods annotations in the JMTE case?

Reply:
Nope, this is a mistake! Nice catch!